client-app/src/desktop/tabs/other/exceptions/ExceptionHandlerPanelModel.js

```diff
@@ -0,0 +1,32 @@
+import {HoistModel, managed} from '@xh/hoist/core';
+import {FormModel} from '@xh/hoist/cmp/form';
+import {XH} from '@xh/hoist/core';
+
+export class ExceptionHandlerModel extends HoistModel {
```

Comment:
File name and class name should be the same.  `ExceptionHandlerModel` is proably fine